src/main/java/com/github/thedeathlycow/moregeodes/features/ModConfiguredFeatures.java

```diff
@@ -213,6 +213,35 @@ public class ModConfiguredFeatures {
                         0.3f
                 )
         ));
+
+        if (FabricLoaderImpl.INSTANCE.isModLoaded("ae2")) {
```

Comment:
use `FabricLoader.getInstance()`, as using implementation classes may break Quilt compatibility

Reply:
I blame IDEA for that one